.github/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,74 @@
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to making participation in our project and
+our community a harassment-free experience for everyone, regardless of age, body
```

Comment:
`harassment` looks like a very strong word in English (https://en.oxforddictionaries.com/definition/harassment). @lsmith77 proposed in the Symfony Slack to use something like this instead:

> I would rephrase this as "make a real effort to not make someone uncomfortable and if you do apologize and adjust your behavior .."

Reply:
It's impossible to not make anybody uncomfortable. People get uncomfortable even if somebody has different opinion, regardless in what area. Therefore I like harassment word better. That words means you are not doing it by accident, but repeatedly.

From [wikipedia](https://en.wikipedia.org/wiki/Harassment):
>  Harassment: It is commonly understood as behavior that disturbs or upsets, and it is characteristically **repetitive**.

I also like this from same page:
> Harassment, under the laws of the United States, is defined as any repeated or continuing un-consented contact that serves no useful purpose beyond creating alarm, annoyance, or emotional distress.